packages/vite/src/node/importGlob.ts

```diff
@@ -94,12 +95,17 @@ export async function transformImportGlob(
       )},`
     } else if (isEager) {
```

Comment:
I think the code is going to be more clear if we change it to
```js
} else {
  const importeeUrl = isCSSRequest(importee) ? `${importee}?used` : importee
  if (isEager) {
    ...
  }
  else {
    ...
  }
}
```
We can avoid repeating the condition, and the main `else` makes it more evident that we are dealing with the non-raw case in both branches.

Reply:
I seem to have found a pattern that when a fragment is reused it should be extracted. 😀

Reply:
ha, not always, early abstraction is sometimes worse than duplicating code. But in this case, the variable applied to both branches when it was non-raw. I moved inside the `else` case, as it is not used in the `raw` branch.